preprocessing-standards.Rmd

```diff
@@ -134,22 +138,23 @@ cwp <- CentWaveParam(ppm = 50,
 
 data <- findChromPeaks(data, param = cwp) 
 pks <- chromPeaks(data)
-```    
-    
-Consider Xanthine
-```{r}
-mz_xan <- 152.03384
-dev <- 0.01
-xan_pos <- data %>%
-    filterRt(rt = c(130, 150)) %>%
-    filterMz(mz = c(mz_xan + 1 - dev, mz_xan + 1 + dev)) %>%
-    filterFile(file = which(data$polarity == "POS"))
+```
+
+We next evaluate the signal for Xanthine. For this we first calculate first its
+monoisotopic mass based on the compound's chemical formula and then the m/z of
+its [M+H]+ ion. 
+
+```{r xanthine-mz}
```

Comment:
this code block is the *standard* way how we calculate m/z values for a compound (based on its formula).